erpnext/crm/doctype/contract/contract_list.js

```diff
@@ -0,0 +1,12 @@
+frappe.listview_settings['Contract'] = {
+    add_fields: ["custom_status"],
+    get_indicator: function (doc) {
+        if (doc.custom_status == "Unsigned") {
+            return [__(doc.custom_status), "orange", "custom_status,=," + doc.custom_status];
```

Comment:
Unsigned to "Red" instead of "Orange" 

Orange suggests that it's "In Progress". Whilst the contract may be signed eventually, and you may call that "In Progress", it's either Signed or Not Signed. So for the sake of clarity, keep it Red.

Reply:
Change to Red